frontend/src/Components/MainContainer/MainContainer.module.css

```diff
@@ -1,37 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik+Puddles&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Pangolin&family=Permanent+Marker');
+@import url('https://fonts.googleapis.com/css2?family=Frank+Ruhl+Libre:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bangers&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lobster+Two&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Shantell+Sans&display=swap');
+
 .mainContainer {
   display: flex;
   flex-direction: column;
   align-content: center;
   flex-wrap: wrap;
+  background-image: url('file:///D:/Jeopardy%20-%20Fdipardy/App/Fdipardy/frontend/src/images/space-galaxy-background-free-vector.webp');
+  background-size: cover;
+  background-position: center;
+  height: 754px;
```

Comment:
Why 754px? For a more responsive design we need to use rem. Also have you tested out zooming in and out of the page - wheter this breaks the design or not?

Reply:
Shelved for laters